lua/lvim/plugins.lua

```diff
@@ -114,10 +113,7 @@ return {
     "hrsh7th/cmp-path",
     commit = commit.cmp_path,
   },
-  {
-    "hrsh7th/cmp-nvim-lua",
-    commit = commit.cmp_nvim_lua,
-  },
+  { "folke/lua-dev.nvim" },
```

Comment:
```suggestion
  { "folke/lua-dev.nvim", commit = commit.lua_dev },
```

And add this at line 15

```lua
  lua_dev = "1933c7e014e69484572b7fa1bf73bc51c42f10f4",
```